docs: replacing pipette o-rings

jwwojak opened the pull request:
# Overview

This PR adds instructions about how to remove and replace pipette o-rings. Instructions apply to 1- and 8-channel pipettes only. Other instruction manuals indicate this user-level procedure is not applicable to 96-channel pipettes, so that instrument is excluded.

Reference: RTC-796

## Test Plan and Hands on Testing

Runs as expected locally. Will have to see in a push.

## Changelog

Adds text modified from the preventative maintenance manual on changing pipette nozzle o-rings. I put this in `cleaning.md`. Please recommend another location if this isn't ideal. I was unsure myself.

Also, some Pipette header levels in this chapter seems too deep (some H4s and H3s). Moved these up a level.

## Review requests

- Is this new content in the right section?
- Are changes to some headers ok (brought some items up one level).
- I used the CSS rule for images. Do these look too big?

## Risk assessment

Low; docs only.

jwwojak committed:
Quick afternoon push

- Added o-ring instructions.

- Added the instruction list style rule to a procedure earlier in the doc and to the o-ring steps.

jwwojak committed:
o-ring PR commit

ecormany commented on docs/flex/docs/system-description/pipettes.md:
```diff
@@ -214,10 +214,14 @@ Opentrons Flex automatically updates pipette firmware to keep it in sync with th
 
 If, for any reason, your pipette firmware and robot software versions get out of sync, you can manually update the firmware in the Opentrons App.
 
+<div class="instruction-list" markdown>
```
We have very similar instructions on the gripper page. If we use this styling here, we should also use it there.

jwwojak replied:
To keep things consistent, I'll remove the class rule to match the gripper.

ecormany reviewed (approved):
Looks great. Just the one note on consistency. Whichever way you go, it can be one more commit on this branch.

jwwojak committed:
Remove class rule, match new images to current

- Use a regular ordered list here with no styling.
- Match pipette removal images to others on page (width=50%).
